src/server/oembed.js

```diff
@@ -0,0 +1,80 @@
+import { Router } from 'express';
+import cors from 'cors';
+import xml from 'xml';
+
+import * as OiceAPI from '../common/api/oice';
+
+import { OICE_VIEW_URL_BASE } from '../common/constants';
+import { ValidationError } from '../common/utils/validation';
+
+const router = Router();
+const { URL } = require('url');
+
+const DEBUG = process.env.NODE_ENV !== 'production';
+
+router.get('/oembed', cors(), async (req, res, next) => {
```

Comment:
if i user `router.use(cors())` instead, would it affect other route that is not inside this router?
if not, should prefer `router.use()`